### ISSUE TITLE
Compiling IScala.jar from the latest source seems to produce errors

### ISSUE THREAD

**Issue by smravuri** [mattpap/IScala#15]
I did a git clone to get the latest code and created IScala.jar using ./sbt, followed by 'compile' and 'assembly'. When I use my locally built IScala.jar with IPython, I get the following error even for simple scala commands. My environment is Mac OSX (one change I had to make was in your 'sbt' script where you had 'openssl md5 -r ...', but in Mac OSX, i had to change it to 'openssl md5 -hex ...'). 

If I instead use your latest 0.1 release IScala.jar (or even build locally using the 0.1 source.zip that you have provided at the release location), it works perfectly fine. I tried debugging by putting breakpoints etc in the Interpreter.scala, but I couldn't quite follow what was going on. Am I doing something wrong? 

In [2]: val x = 1
scala.ScalaReflectionException: object $line10.$display.$iw not found.
    scala.reflect.internal.Mirrors$RootsBase.staticModule(Mirrors.scala:162)
    scala.reflect.internal.Mirrors$RootsBase.staticModule(Mirrors.scala:22)
    org.refptr.iscala.Interpreter.runCode(Interpreter.scala:118)
    org.refptr.iscala.Interpreter.display(Interpreter.scala:162)
    org.refptr.iscala.Interpreter$$anonfun$10.apply(Interpreter.scala:206)
    org.refptr.iscala.Interpreter$$anonfun$10.apply(Interpreter.scala:206)
    org.refptr.iscala.Interpreter.withException(Interpreter.scala:97)
    org.refptr.iscala.Interpreter.loadAndRunReq(Interpreter.scala:206)
    org.refptr.iscala.Interpreter$$anonfun$interpret$1.apply(Interpreter.scala:236)
    org.refptr.iscala.Interpreter$$anonfun$interpret$1.apply(Interpreter.scala:236)
    org.refptr.iscala.Runner$Execution$$anonfun$1.apply$mcV$sp(Runner.scala:28)
    org.refptr.iscala.IOUtil$$anon$2.run(Util.scala:21)
    java.lang.Thread.run(Thread.java:744)

**Comment by mattpap** [comment]
I'm guessing that your osx is installed into an case-insensitive file system? That would explain this error, because what you see is just effect of failed compilation of display object. In display module, there is `Latex` type class and `LaTeX` case class which will clash on such file system. Well, naming things is one of the hardest issues in software development. I will have to do some renaming to fix this issue.

**Comment by mattpap** [comment]
> but in Mac OSX, i had to change it to 'openssl md5 -hex ...'

I will apply this change to `sbt`. I recall hearing this issue from a user of one of my other Scala projects.

**Comment by smravuri** [comment]
Ah.. thanks. I did not know that my osx installation has case-insensitive file system. I googled around this and found several people pointing out that applications like Photoshop and a few others don't support case-sensitive filesystems. So I suppose, I will have to live with a case-insensitive system. 

As a quick workaround, I renamed LaTeX case class locally (to, say, LaTeXFormat). I recompiled and deployed. It works fine now. I can wait for a fix from you whenever you get around to this.

Thanks for your help.
